actor/kvcounter-tinygo/kvcounter_tinygo.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"strconv"
+
+	"github.com/wasmcloud/actor-tinygo"
+	httpserver "github.com/wasmcloud/interfaces/httpserver/tinygo"
+	keyvalue "github.com/wasmcloud/interfaces/keyvalue/tinygo"
+)
+
+func main() {
+	me := KvcounterTinygo{}
```

Comment:
not sure if it matters, but we could drop this to a one-liner

Reply:
Done!